nhs_context_logging/logger.py

```diff
@@ -383,7 +383,7 @@ def get_method_name(func, *args, **kwargs):
     else:
         method_name = func.__name__
 
-    if app_logger.config.prepend_module_name:
+    if hasattr(app_logger, "config") and app_logger.config.prepend_module_name:
```

Comment:
think this is not the right approach, we should ensure that the config is always present

Reply:
i've changed it to set the default in the __init__